File: cpp/makanan.cpp
#pragma once
#include <bits/stdc++.h>
using namespace std;

class makanan 
{
private:
    string nama;
    string jenis;
public:
    makanan() {}
    makanan(string nama, string jenis) 
    {
        this->nama = nama;
        this->jenis = jenis;
    }

    // Setter
    void setNama(string nama) { this->nama = nama; }
    void setJenis(string jenis) { this->jenis = jenis; }

    // Getter
    string getNama() { return nama; }
    string getJenis() { return jenis; }

    void print()
    {
        cout << "Makanan            : " << getNama() << " (" << getJenis() << ")" << endl;
    }
};

File: java/Makanan.java
public class Makanan {
    private String nama;
    private String jenis;

    public Makanan() {}

    public Makanan(String nama, String jenis) {
        this.nama = nama;
        this.jenis = jenis;
    }

    // Setter
    public void setNama(String nama) { this.nama = nama; }
    public void setJenis(String jenis) { this.jenis = jenis; }

    // Getter
    public String getNama() { return nama; }
    public String getJenis() { return jenis; }

    public void print() {
        System.out.println("Makanan            : " + getNama() + " (" + getJenis() + ")");
    }
}